lib/invoice_item.rb

```diff
@@ -1,7 +1,8 @@
+require 'pry'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.